plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -294,7 +294,6 @@ public void testFilteringForTablesAndColumns()
     }
 
     @Test
-    @Override
     public void testLimitPushDown()
```

Comment:
does `testLimit` pass for druid without an override?

Reply:
Yes it does.